src/test/java/io/lettuce/core/pubsub/PubSubCommandBuilderUnitTests.java

```diff
@@ -124,4 +126,15 @@ void unsubscribe() {
         assertEquals( "key<channelPattern>",  command.getArgs().toCommandString());
         assertInstanceOf(PubSubOutput.class, command.getOutput());
     }
+
+    @Test
+    void ssubscribe() {
```

Comment:
I don't know why this class keeps using JUnit assertions. It would be good to rewrite it to AssertJ (there are IDE plugins that can do a majority of the conversion).

Reply:
will check for all within another pr